### ISSUE TITLE
I need a PR approval

### ISSUE THREAD

**Issue by deeksha345** [Azure/azure-cli#24075]
I need an approval for a PR I created for cli changes - [K8s extension/release 1.3.6 by deeksha345 · Pull Request #5406 · Azure/azure-cli-extensions (github.com)](https://github.com/Azure/azure-cli-extensions/pull/5406)
The people who are assigned to this PR are OOF till Oct 7th but I have a customer waiting on this change and so would like to get this merged ASAP.

**Comment by yonzhan** [comment]
route to CXP team

**Comment by navba-MSFT** [comment]
@deeksha345 Thanks for reaching out to us and reporting this issue. I see that you have closed [this](https://github.com/Azure/azure-cli-extensions/pull/5406) PR now.  Do you still need any assistance on that PR or not ? Awaiting your reply.

**Comment by deeksha345** [comment]
Hi yes sorry I do not need that PR to be merged I will be opening another PR with more recent updates which will need approval!

**Comment by navba-MSFT** [comment]
@deeksha345 Once you file the PR, please let us know, I will reach out to the concerned assignee for the review and approval.

**Comment by deeksha345** [comment]
@navba-MSFT this is the new PR with the latest changes - https://github.com/Azure/azure-cli-extensions/pull/5424
